docs/examples/ex_dmsp_methods.rst

```diff
@@ -0,0 +1,86 @@
+.. _ex-dmsp-methods:
+
+DMSP Ion Drifts at High Latitudes
+=================================
+
+Plasma motion measured by the Ion Velocity Meter (IVM) onboard DMSP is natively
+presented in spacecraft coordinates and does not include any transformations
+of those directions into published coordinate systems. We present an example
+on how to plot measured ion drifts at high latitudes using supporting methods
+in :py:meth:`pysatMadrigal.instruments.methods.dmsp`.
+
+.. code::
+
+   import datetime as dt
+   import matplotlib.pyplot as plt
+   import numpy as np
+   import numpy.ma as ma
+
+   import pysat
+   import pysatMadrigal as pysat_mad
+
+   fdate = dt.datetime(2011, 8, 15)
+   ivm = pysat.Instrument('dmsp', 'ivm', tag='utd', inst_id='f15')
+
+   # Smooth ion drifts to only report the most robust of measurements.
+   ivm.custom_attach(pysat_mad.instruments.methods.dmsp.smooth_ram_drifts,
+                     kwargs={'rpa_flag_key': 'rpa_flag_ut'})
+
+   # Add unit vectors relating presumed satellite orientation along
+   # cartesian and polar directions.
+   ivm.custom_attach(pysat_mad.instruments.methods.dmsp.add_drift_unit_vectors)
+
+   # Express ion drifts measured in the satellite frame along the added
+   # high latitude unit vectors.
+   ivm.custom_attach(pysat_mad.instruments.methods.dmsp.add_drifts_polar_cap_x_y,
+                     kwargs={'rpa_flag_key': 'rpa_flag_ut'})
+
+   # Download data if not already on system
+   if fdate not in ivm.files.files.index:
+       ivm.download(fdate)
+
+   # Load data
+   ivm.load(date=fdate)
+
+   # Plot multiple orbital passes across polar cap
+   f = plt.figure()
```

Comment:
```suggestion
   fig = plt.figure()
```

Recommend using more descriptive variable names and saving the single letter variable names for indices.